it/uk/gov/hmrc/traderservices/services/MongoDBCachedJourneyServiceISpec.scala

```diff
@@ -1,28 +1,20 @@
 package uk.gov.hmrc.traderservices.services
 
-import play.api.libs.json.Format
-import uk.gov.hmrc.traderservices.repository.CacheRepository
 import akka.actor.ActorSystem
+import com.typesafe.config.Config
+import org.mongodb.scala.{MongoClient, MongoDatabase}
+import play.api.libs.json._
+import uk.gov.hmrc.mongo.{CurrentTimestampSupport, MongoComponent}
+import uk.gov.hmrc.mongo.cache.{CacheItem, DataKey}
+import uk.gov.hmrc.traderservices.journeys.Transition
+import uk.gov.hmrc.traderservices.repository.CacheRepository
 import uk.gov.hmrc.traderservices.support.AppISpec
-import uk.gov.hmrc.play.fsm.JourneyModel
+
+import java.time.Instant
 import java.util.UUID
 import scala.concurrent.ExecutionContext.Implicits.global
-import uk.gov.hmrc.crypto.ApplicationCrypto
-import uk.gov.hmrc.play.fsm.PersistentJourneyService
-import scala.concurrent.Future
-import play.api.libs.json.{Format, JsError, JsNumber, JsSuccess, Reads, Writes}
-import scala.concurrent.ExecutionContext
-import uk.gov.hmrc.mongo.MongoComponent
+import scala.concurrent.{ExecutionContext, Future}
 import scala.concurrent.duration.Duration
-import org.mongodb.scala.MongoClient
-import org.mongodb.scala.MongoDatabase
-import uk.gov.hmrc.mongo.CurrentTimestampSupport
-import uk.gov.hmrc.mongo.cache.CacheItem
-import uk.gov.hmrc.mongo.cache.DataKey
-import java.time.Instant
-import play.api.libs.json.Json
-import play.api.libs.json.JsValue
-import play.api.libs.json.JsString
 
 class MongoDBCachedJourneyServiceISpec extends MongoDBCachedJourneyServiceISpecSetup {
```

Comment:
As the file MongoDBCachedJourneyService.scala file was deleted, please consider using the correct name of the file and replace of the word MongoDBCachedJourneyService with the correct one.

Reply:
Done.